class.jetpack-options.php

```diff
@@ -123,6 +123,30 @@ public static function is_valid( $name, $group = null ) {
 	}
 
 	/**
+	 * Checks if an option must be saved for the whole network in WP Multisite
+	 *
+	 * @param $option_name
+	 *
+	 * @return bool
+	 */
+	public static function is_network_option( $option_name ) {
+		if ( ! is_multisite() ) {
+			return false;
+		}
+		return in_array( $option_name, self::get_network_options() );
+	}
+
+	/**
+	 * Return a list of options to be saved in wp_sitemeta table
+	 * in Network sites (wp_options otherwise)
+	 *
+	 * @return array
+	 */
+	public static function get_network_options() {
```

Comment:
I changed this in my last commit. I've removed ```get_network_options()``` and add a third case inside ```Jeptack_Options::get_option_names()``` so you can get the network options list with ```Jetpack_Options::get_option_names('network')```